package.json

```diff
@@ -90,25 +90,19 @@
       "/.module-cache/",
       "/react/build/"
     ],
-    "persistModuleRegistryBetweenSpecs": true,
```

Comment:
Jest upgrade path was so good. For each option that was deprecated, there was a big blob of text saying why and what was the new one I needed to use.